src/call.go

```diff
@@ -235,6 +235,15 @@ func (c *Call) iceCandidateHandler(candidate *webrtc.ICECandidate) {
 
 	jsonCandidate := candidate.ToJSON()
 
+	callCandidate := event.CallCandidate{
+		Candidate:     jsonCandidate.Candidate,
+		SDPMLineIndex: int(*jsonCandidate.SDPMLineIndex),
+		SDPMID:        *jsonCandidate.SDPMid,
+	}
+	if jsonCandidate.UsernameFragment != nil {
+		callCandidate.UsernameFragment = *jsonCandidate.UsernameFragment
```

Comment:
also the password if we're doing this

Reply:
![2022-08-16_15-31](https://user-images.githubusercontent.com/25768714/184892105-d2ecac18-4fa2-43ab-bf5d-23190e3f462d.png)

There seems to be no way to do that? EW also doesn't seem to be sending it

Reply:
Right yes, sorry - I got confused. The ice ufrag / password are just in the SDP rather than the candidates, so they don't need to be here at all. I'm not sure why it's on `IceCandidateInit`.